src/codeu/chat/common/NetworkCode.java

```diff
@@ -36,4 +36,6 @@
       RELAY_READ_RESPONSE = 28,
       RELAY_WRITE_REQUEST = 29,
       RELAY_WRITE_RESPONSE = 30;
```

Comment:
I'm not sure if this would compile, because the Server ints don't have the typical <int name = value> format. Instead, I think the Relay_Write_Response  and the Server_Info_Request should end in a comma, similar to the formatting of the others. At least, that's how I did it.